src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx

```diff
@@ -192,7 +204,7 @@ const AuctionResultsContainer: React.FC<AuctionResultsProps> = ({
             toggleMobileActionSheet={toggleMobileActionSheet}
           />
 
-          <Spacer mt={3} />
+          <Spacer mt={["2", "0"]} />
```

Comment:
**Note:** We need more space on mobile.

Reply:
Shouldn't need to wrap these values in strings per rest of file.